src/components/top-tab-bar/TopTabBar.tsx

```diff
@@ -47,9 +47,9 @@ const createTabBarStyles = (themeColors: ThemeColors) =>
       position: 'relative',
       zIndex: 100,
       shadowColor: themeColors.neutralDark1,
-      shadowOpacity: 0.15,
+      shadowOpacity: 0.12,
```

Comment:
We should probably add `elevation` too so there is a shadow on android for now

Reply:
for some context about a potential future with this! https://linear.app/audius/issue/AUD-1276/research-cross-platform-box-shadow